package.json

```diff
@@ -80,6 +80,7 @@
     "react-window": "1.8.5",
     "redux": "4.0.5",
     "redux-thunk": "2.3.0",
+    "reselect": "^4.0.0",
```

Comment:
Don't forget to pin this version, as we do with the others 👍

Reply:
Upps, thanks!